script/Create2Factory.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/utils/Create2.sol";
+
+contract Create2Factory {
+    event Deployed(address addr, address deployer, bytes32 bytecode_hash, bytes32 salt);
```

Comment:
Do we want indexes on this event so  can easily fetch in future?

Reply:
Good point! I missed! i can add it on next deployment! Thanks